README.md

```diff
@@ -68,6 +68,10 @@ configuration file.
 |                                     **_doom_one_**<br>[source](https://github.com/hlissner/emacs-doom-themes)                                     |              ![doom_one](images/doom_one.png)              |
 |                                                **_dracula_**<br>[source](https://draculatheme.com)                                                |               ![dracula](images/dracula.png)               |
 |                                                **_dracula_plus_**<br>[source](https://draculatheme.com)                                           |               ![dracula_plus](images/dracula_plus.png)     |
+|                                                **_enfocado_nature_dark_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)          |![enfocado_nature_dark](images/enfocado_nature_dark.png)    |
+|                                                **_enfocado_nature_light_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)         |![enfocado_nature_light](images/enfocado_nature_light.png)  |
+|                                                **_enfocado_neon_dark_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)            |![enfocado_neon_dark](images/enfocado_neon_dark.png)        |
+|                                                **_enfocado_neon_light_**<br>[source](https://github.com/wuelnerdotexe/vim-enfocado.git)           |![enfocado_neon_light](images/enfocado_neon_light.png)      |
```

Comment:
Where did you get the names of these colorschemes from? From the theme's readme it seems like both nature and neon have the same colors, but they just use them for different syntax tokens (which is irrelevant for Alacritty).

Reply:
Considering that as far as I can tell the enfocado theme just has different colors for different syntax tokens with the two different themes, I think it makes sense to only have on enfocado dark/light theme, not two of each.